### DISCUSSION TITLE
Фильтрация стран (убрать зарубежье)

### DISCUSSION THREAD

**Discussion by idia** [cdek-it/widget#82] [Ideas]
Уже год ждем функционала, чтобы была возможность при вводе названия города показывать только города по определенной стране.
Например, ИМ отправляет только в РФ.
При этом при вводе САНКТ выводится все, что угодно (и Бразилия, и Франция), но только не Санкт-Петербург, Россия :(

Можно как-то решить вопрос в текущем функционале виджета?

**Comment by Andrew100500** [comment]
Так задача уже решена в 4 версии которая вышла в конце 2 квартала 2024 года.  Что не видели?  Вот и я не вижу.  А разработчики упорно работают над третьей версией место котророй в конечном итоге в  мусорном ведре.

**Reply by idia** [reply]
Кажется, там один разработчик. Он же тимлид, он же фронт, он же проджект, он же тестировщик, он же продакт. Иначе не обьяснить такие "темпы" разработки. И СДЭК при этом себя называет IT-платформой :(

**Reply by Andrew100500** [reply]
У них по API по запросу по индексу Вышний Волочек, передается с буквой е. Далее я беру название и вставляют в виджет карты и нефига нет такого города.  Потому что в виджете Вышний Волочёк забит через ё.  Ну то что е и ё в русском языке равнозначные и попроавить поиск с учетом этого дело пары минут, как я думал. Но нет. IT отдел ответил, что гужно собирать всепсоюзный съеззд менеджеров, на котором будет рещаться менять е на ё или не менять и этим должен заняться я, потому что им похрнен.  Вот так работает it отдел. Они ошибки не исправляют, они их отчаянно защищают.  Я у себя решил вопрос костылем.  И таких населенных пунктов далеко не один.
